### ISSUE TITLE
Mini-Player: Progress gets stuck

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-ios#2472]
### Description

We've noticed an issue where the episode progress is getting stuck while the episode plays, but it updates when you expand the player. It seems to be related to Intelligent Playback Resumption as well. 

The devs mention the following during this internal discussion: p1731964106513389/1731019987.528549-slack-C02A333D8LQ

> I think it happens when the app is launched and an episode is in the queue. The mini player has a wrong progress applied and doesn’t update unless you put it in full screen

### Step-by-step reproduction instructions

1. Enable Intelligent Playback Resumption
2. Listen partially to an episode and close Pocket Casts
3. Reopen Pocket Casts and resume playback via the mini-player
4. You should notice the progress is wrong or stuck, and the player needs to be opened for it to be fixed

### Screenshots or screen recording

https://github.com/user-attachments/assets/2c12b1a0-058b-4afd-b8b3-5fc9265aa135

### Did you search for existing bug reports?

- [x] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

App Version: 7.75
Device: iPhone17,1
OS: 18.1

**Comment by rviljoen** [comment]
@joashrajin Think this is a duplicate of https://github.com/Automattic/pocket-casts-ios/issues/1873

**Comment by joashrajin** [comment]
@rviljoen possible, but the reproduction steps aren't very clear in that report 🤔

**Comment by rviljoen** [comment]
> @rviljoen possible, but the reproduction steps aren't very clear in that report 🤔 

Agreed, but was more thinking of merging the two issues so that there are not duplicate issues hanging around.

**Comment by SergioEstevao** [comment]
I was unable to reproduce this on 7.78. Can someone confirm it?

**Comment by rviljoen** [comment]
I have witnessed this while using the app day to day, but have never been able to reproduce this during debugging 😬

**Comment by danielebogo** [comment]
@SergioEstevao this usually happens on the startup. Running the app and play the episode from the MiniPlayer without doing anything else
